javascript/builtins/String.json

```
@@ -3182,9 +3182,15 @@
               "ie": {
                 "version_added": false
               },
-              "nodejs": {
-                "version_added": "10.0.0"
-              },
+              "nodejs": [
+                {
+                  "version_added": "10.0.0"
+                },
+                {
+                  "version_added": true,
```

From the test results from [the page linked in the issue](https://node.green/#ES2019-features-string-trimming-String-prototype-trimRight), it looks like these first appeared in 0.12:

```suggestion
                  "version_added": "0.12",
```